internal/controller/controllers/clusterdeployments_controller.go

```diff
@@ -1464,6 +1469,10 @@ func CreateClusterParams(clusterDeployment *hivev1.ClusterDeployment, clusterIns
 		clusterParams.OlmOperators = olmOperators
 	}
 
+	if label := clusterInstall.GetLabels()[hiveext.ClusterConsumerLabel]; label != "" {
```

Comment:
nit: we should test for the key instead of relying on the 0 value, same in the update function

Reply:
Modified! How does it look?